### PR TITLE
stm32cube: Add stm32l5 package (v1.2.0)

### PR DESCRIPTION
**Pull request by erwango**
Introduce STM32Cube package for STM32L5xx series (v1.2.0)
    
URL:
       https://github.com/STMicroelectronics/STM32CubeL5
Commit:
       2ca47b7feebb1852e49ff34752201999fe01eb53
Maintained-by:
       External 
License:
       BSD-3-Clause
License Link:
       https://opensource.org/licenses/BSD-3-Clause

### CODE REVIEW

**Comment by erwango** [comment]
^^@FRASTM, can you review ?

File: stm32cube/stm32l5xx/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2017 Erwin Rol <erwin@erwinrol.com>
+# Copyright (c) 2020 STMicroelectronics.
```
Comment (FRASTM) [inline stm32cube/stm32l5xx/CMakeLists.txt]: # Copyright (c) 2020 STMicroelectronics

Comment (erwango) [reply stm32cube/stm32l5xx/CMakeLists.txt]: @FRASTM , done

**Comment by erwango** [comment]
@galak ready for merge, if you're fine with it (tested in https://github.com/zephyrproject-rtos/zephyr/pull/23551)

**Pull request merged**